src/modules/ens/ens.service.ts

```diff
@@ -159,7 +159,7 @@ export class EnsService {
     owner?: string;
   }) {
     try {
-      if (!owner  || owner !== emptyAddress) {// prevents attempt to resync after namepsace deregisteration
+      if (owner !== emptyAddress) {// prevents attempt to resync after namepsace deregisteration
```

Comment:
Won't it prevent deletion of the domain? To delete domain we set its owner to empty address, but this check just stops processing instead of removing domain from db. Also since e2e testing already configured we could make use of it here